Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -83,6 +83,15 @@ public extension UIConfig {
         public var galleryImageBackground: UIColor = .streamWhiteSmoke
         public var galleryUploadingOverlayBackground: UIColor = UIColor.black.withAlphaComponent(0.5)
         public var galleryUploadingProgressBackground: UIColor = UIColor.black.withAlphaComponent(0.6)
+        public var messageActionIconDefaultTint: UIColor = .streamGray
+        public var messageActionIconTints: [ChatMessageActionItem.Name: UIColor] = [
+            .delete: .streamAccentRed,
+            .resend: .streamAccentBlue
```

Comment:
"All blue must come from `tintColor`"

Reply:
`tintColor` is used in f8737feaec6515316f4ba45ef12033ce0ba07658